### PR TITLE
Fix the issues with the package selection page

### PR DESCRIPTION
**Pull request by tecimovic**
0. Properly shortcut the config page if there is only one of each package type.

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -48,6 +46,17 @@ Cypress.Commands.add('rendererApi', (...args) => {
...
 
 Cypress.Commands.add('setZclProperties', () => {
-  cy.get('[data-test="gen-template"]').click()
-  cy.get('[data-test="login-submit"]').click()
-})
\ No newline at end of file
+  cy.get('body').then(($body) => {
+    // The above doesn't work. It does find the buttons, even
+    // if they are hidden, obviously....
+    /*if ($body.find('[data-test="gen-template"]').length) {
```
Comment (brdandu) [inline cypress/support/commands.js:52]: Do we need to update the tests to account for this behavior. With single package and without single package?

File: src/App.vue [hunk 2]
```diff
@@ -113,8 +112,11 @@ export default {
     getAppData() {
       if (this.$serverGet != null) {
         this.$serverGet(restApi.uri.uiOptions).then((res) => {
-          this.$store.commit('zap/updateIsProfileIdShown', res.data.showProfileId)
-          console.log(res.data);
+          this.$store.commit(
+            'zap/updateIsProfileIdShown',
+            res.data.showProfileId
+          )
+          console.log(res.data)
```
Comment (brdandu) [inline src/App.vue:119]: Get rid of console.log here?

File: src/App.vue [hunk 3]
```diff
@@ -187,13 +189,13 @@ export default {
...
-        this.$router.push({path: '/login'})
+      if (val != true) {
+        this.$router.push({ path: '/login' })
       } else {
         this.$router.push({ path: '/' })
         this.getAppData()
       }
-    }
-  }
+    },
```
Comment (brdandu) [inline src/App.vue:198]: Not sure why the additional , here comes from.

**Review by paulr34 — APPROVED** [review]

File: cypress/support/commands.js (hunk 1 above)
Comment (tecimovic) [reply cypress/support/commands.js:52]: We need to do something. tried the above and failed. Not sure what else to try. I'll have to study up cypress a bit more.

File: src/App.vue (hunk 2 above)
Comment (tecimovic) [reply src/App.vue:119]: yeah.

File: src/App.vue (hunk 3 above)
Comment (tecimovic) [reply src/App.vue:197]: Auto-formatter. I'll let it do its thing.

**Pull request merged**